README.md

```diff
@@ -46,6 +46,13 @@ If you want to start from an empty database, delete the `db` folder.
 
 Note: during the ingest process no data leaves your local environment. You could ingest without an internet connection.
 
+Supported file types:
```

Comment:
Could you move this to the beginning of the "Instructions for ingesting your own dataset" section? And probably modify the first phrase where it says "put any and all of your .txt, .pdf or .csv files..." so the whole section is aligned. 

Id suggest:

Put any and all of your files into the source_documents directory

Supported file types:
- txt
...

Reply:
Done